test/hackney_stream_tests.erl

```diff
@@ -53,4 +76,5 @@ receive_response(Ref, Dict0) ->
         {hackney_response, Ref, Bin} ->
             Dict1 = orddict:append(body, Bin, Dict0),
             receive_response(Ref, Dict1)
+    after 5000 -> {error, timeout}
```

Comment:
i think i would make it configurable.

Reply:
Added a parameter to make it configurable by test case